tests/fetch/stream_file.cpp

```diff
@@ -28,15 +28,15 @@ int main()
 #endif
   };
   attr.onprogress = [](emscripten_fetch_t *fetch) {
-    assert(fetch->data != 0);
-    assert(fetch->numBytes > 0);
-    assert(fetch->dataOffset + fetch->numBytes <= fetch->totalBytes);
-    assert(fetch->totalBytes <= 134217728);
     printf("Downloading.. %.2f%s complete. Received chunk [%llu, %llu[\n", 
       (fetch->totalBytes > 0) ? ((fetch->dataOffset + fetch->numBytes) * 100.0 / fetch->totalBytes) : (double)(fetch->dataOffset + fetch->numBytes),
       (fetch->totalBytes > 0) ? "%" : " bytes",
       fetch->dataOffset,
       fetch->dataOffset + fetch->numBytes);
+    assert(fetch->data != 0);
```

Comment:
Indentation?

Reply:
Oh, yep, that'd do it. I got confused cause I thought the closure ended for... some reason. Carry on!